Article/Article.js

```diff
@@ -112,3 +112,106 @@ const data = [
   Step 5: Add a new article to the array. Make sure it is in the same format as the others. Refresh the page to see the new article.
 
 */
+
+
+
+ /* <div class="article">
+    <h2>{title of the article}</h2>
+    <p class="date">{date of the article}</p>
+
+    {three separate paragraph elements}
+
+    <span class='expandButton'></span>
+  </div>
+
+  Hint: You will need to use createElement more than once here!*/
+  
+  function createComponent({title,date,firstParagraph,secondParagraph,thirdParagraph}) {
+      const div = document.createElement('div');
+      div.setAttribute('class','article');
+
+      const h2 = document.createElement('h2');
+      h2.textContent = title;
+
+      const p = document.createElement('p');
+      p.setAttribute('class','date');
+      p.textContent = date;
+
+      const p1 = document.createElement('p');
+      p1.textContent =  firstParagraph;
+
+      const p2 = document.createElement('p');
+      p2.textContent =  secondParagraph;
+
+      const p3 = document.createElement('p');
+      p3.textContent =  thirdParagraph;
+
+      const span = document.createElement('span');
+      span.setAttribute('class','expandButton');
+      span.textContent = "Toggle";
+      span.style.textAlign = "center";
+
+      
+    //   span.
+
+      span.addEventListener('click', e => {
+          document.querySelector('.article')
+          .classList.toggle('article-open')
+      })
```

Comment:
@osammy this only affects the first article on the page, therefore `Toggle` button on other articles is not going to work. Any ideas for proper solution? 🙄🔥